### PR TITLE
[js][BiDi] Browsing context commands

### PR DESCRIPTION
**Pull request by TamsilAmani**
### Description

Add BrowsingContext module commands based on the spec.
Refer to [BiDi spec doc](https://docs.google.com/document/d/1dCd8Y2PYaR5mOGSmNTwllEHNmFqegfoGkP-TCKvPzSU/edit?usp=sharing) for details.

### Motivation and Context

Support BrowsingContext commands based on the W3C BiDi spec. Currently, the commands supported by browsers are made public. Refer https://wpt.fyi/results/webdriver/tests/bidi/browsing_context?label=experimental&label=master&aligned&view=subtest. As browsers implement the rest of the commands, we can make them public (adding methods will not be a breaking change).

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: javascript/node/selenium-webdriver/test/bidi/bidi_test.js [hunk 1]
```diff
@@ -104,6 +105,110 @@ suite(
         await inspector.close()
       })
     })
+
+    describe('Browsing Context', function (){
```
Comment (pujagani) [inline javascript/node/selenium-webdriver/test/bidi/bidi_test.js]: Nit: Add space after function()

**Comment by pujagani** [comment]
LGTM from BiDi spec perspective.

**Comment by harsha509** [comment]
@TamsilAmani ,

Can you do `npm run lint` to fix any lint issues!

File: javascript/node/selenium-webdriver/test/bidi/bidi_test.js (hunk 1 above)
Comment (TamsilAmani) [reply javascript/node/selenium-webdriver/test/bidi/bidi_test.js]: Change incorporated.

**Comment by TamsilAmani** [comment]
@harsha509 I ran `npm run lint` and errors were present in 3 files but the changes were not made by me. So I didn't change them.

**Review by harsha509 — APPROVED** [review]

**Pull request merged**